bindings/node/src/Version.cpp

```diff
@@ -0,0 +1,140 @@
+#include <arbiter/Version.h>
+#include "Version.h"
+
+namespace ArbiterNodeBindings {
+
+using v8::Context;
+using v8::Function;
+using v8::FunctionCallbackInfo;
+using v8::FunctionTemplate;
+using v8::Isolate;
+using v8::Just;
+using v8::Local;
+using v8::Maybe;
+using v8::Nothing;
+using v8::Number;
+using v8::Object;
+using v8::Persistent;
+using v8::String;
+using v8::Value;
+
+Persistent<Function> SemanticVersion::constructor;
+
+SemanticVersion::SemanticVersion(unsigned major, unsigned minor, unsigned patch, Maybe<char *> prereleaseVersion, Maybe<char *> buildMetadata) {
+  _semanticVersion = ArbiterCreateSemanticVersion(
+    major,
+    minor,
+    patch,
+    (prereleaseVersion.IsJust() ? prereleaseVersion.FromJust() : NULL),
+    (buildMetadata.IsJust() ? buildMetadata.FromJust() : NULL)
+  );
+}
+
+SemanticVersion::SemanticVersion(const char *string) {
+  _semanticVersion = ArbiterCreateSemanticVersionFromString(string);
+}
+
+SemanticVersion::~SemanticVersion() {
+}
+
+void SemanticVersion::Init(Isolate* isolate) {
+  // Prepare constructor template
+  Local<FunctionTemplate> tpl = FunctionTemplate::New(isolate, New);
+  tpl->SetClassName(String::NewFromUtf8(isolate, "SemanticVersion"));
```

Comment:
Ah, I see—this is referring to `SemanticVersion::New`, not some magic symbol. In that case, nevermind, since there isn't much which avoids the C++ class.